ksql-engine/src/main/java/io/confluent/ksql/function/Blacklister.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.function;
+
+public interface Blacklister {
```

Comment:
`@FunctionalInterface`?  Also, why not just `Predicate<String>`? Just after a more descriptive type?

Personally, `Blacklister` suggests something that 'can black list things'.  Where as this is just a known blacklist that we're checking against, so maybe just `BlackList`?

Reply:
Yeah it could be `Blacklist`. But yeah, `Predicate` is fine, just trying to describe what it actually is